cluster/gce/gci/configure.sh

```diff
@@ -276,6 +277,17 @@ function install-exec-auth-plugin {
   mv "${KUBE_HOME}/gke-exec-auth-plugin" "${KUBE_BIN}"
 }
 
+function download-conntrack-binaries {
+  if [ -e "${KUBE_BIN}/conntrack" ]; then
```

Comment:
Just out of curiosity, if there a possibility the binary is pre-install on the node so that this line of check is required?

Reply:
Only if the script ran before and downloaded it here.